tests/chainerx_tests/unit_tests/routines_tests/test_activation.py

```diff
@@ -75,6 +76,53 @@ def forward_xp(self, inputs, xp):
 ]
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize(*(
+    chainer.testing.product([
+        chainer.testing.from_pytest_parameterize(
+            'shape', [
+                (2, 2),
+                (3, 3, 3),
+                (5, 5, 5),
+                (4, 1, 2, 4)
+            ]),
+        chainer.testing.from_pytest_parameterize(
+            'in_dtypes,out_dtype', math_utils.in_out_dtypes_math_functions)
+    ])
+))
+class TestLeakyRelu(op_utils.ChainerOpTest):
```

Comment:
Could you also parameterize over special shapes and values as in e.g. the ReLU test below?